Tests/DatadogTests/Datadog/Tracing/Autoinstrumentation/FirstPartyHostsTests.swift

```diff
@@ -0,0 +1,137 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import Datadog
+
+class FirstPartyHostsTests: XCTestCase {
+    let hostsDictionary: [String: Set<TracingHeaderType>] = [
+        "http://first-party.com/": .init(arrayLiteral: .w3c, .b3s),
+        "https://first-party.com/": .init(arrayLiteral: .w3c, .b3s),
```

Comment:
Hint: array literals do automatically convert into dictionaries, so we can make it vastly shorter in all tests:
```suggestion
        "http://first-party.com/": [.w3c, .b3s],
        "https://first-party.com/": [.w3c, .b3s],
        // ...
```